### PR TITLE
(fixes 7582) stops Big 4 and MS from going less than 0

### PR DESCRIPTION
**Pull request by captianobi**
(fixes 7582)

### CODE REVIEW

File: lib/engine/step/g_1846/draft_distribution.rb [hunk 1]
```diff
@@ -90,7 +90,15 @@ def process_pass(action)
             "to #{@game.format_currency(new_value)}"
 
           @round.next_entity_index!
-          return if new_value.positive?
+          if company.name == 'Big 4'
```
Comment (tobymao) [inline lib/engine/step/g_1846/draft_distribution.rb:93]: case company.id
when 'BIG4'
return if new_value >= 60
when 'MS' 
return if new_value >= 80
else
return if new_value.positive?

**Comment by jrimmer** [comment]
Nice work, @captianobi !

File: lib/engine/step/g_1846/draft_distribution.rb [hunk 2]
```diff
@@ -90,14 +90,13 @@ def process_pass(action)
...
-          elsif new_value.positive?
-            return true
+          case company.id
+          when 'Big 4'
+            return if new_value >= 60
+          when 'MS'
+            return if new_value >= 80
           else
-            return false
+            return new_value.positive?
```
Comment (tobymao) [inline lib/engine/step/g_1846/draft_distribution.rb:99]: this is not correct, it should be return if new_value.positive?

Comment (captianobi) [reply lib/engine/step/g_1846/draft_distribution.rb]: All fixed

**Review by tobymao — APPROVED** [review]

**Pull request merged**